Kulumbegov A.L./[READY].txt

```diff
@@ -0,0 +1 @@
+Student of eltech, 17 y.o. tryed Java
```

Comment:
There should be no dots in directory name
Also do not use pull request prefix in file names